src/commands/transaction/list.ts

```diff
@@ -120,11 +148,16 @@ export class ListCommand extends QuestionCommand<PromptAnswers> {
   }
 
   async run(): Promise<void> {
+    const concise = this.opts.concise;
     await this._promptForProperty();
     const response = await this._sendRequest();
     if (response.isError) {
       return;
     }
-    displayTransactions(response);
+    if (!concise) {
```

Comment:
I think we can just do `if (!this.opts.concise)`.

Reply:
Absolutely!